pkg/constants/const.go

```diff
@@ -0,0 +1,31 @@
+package constants
+
+import "time"
+
+type Function int
+
+const (
+	CreateIndex Function = 0
+	UpdateIndex Function = 1
+	DeleteIndex Function = 2
```

Comment:
The names are confusing, cause creating index has other meaning in elastic...
```suggestion
	CreateBook Function = 0
	UpdateBook Function = 1
	DeleteBook Function = 2
```

Reply:
I think we need generalization for enum const. What about the name:
Do<>Index, do can actually help us here because it is a call to action. DoCreateIndex -> we need the createIndex

Reply:
Related to my comment about using the index word as it is confusing...
That's why I suggest to stay with the general `<action>Book`